plugins/crowdin/access_token.go

```diff
@@ -0,0 +1,63 @@
+package crowdin
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.AccessToken,
+		DocsURL: sdk.URL("https://developer.crowdin.com/configuration-file/#api-credentials-from-environment-variables"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Crowdin.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 80,
```

Comment:
It looks like the "Personal Access Token" generated from [this page](https://crowdin.com/settings#api-key) is of 80 characters, but the "API Token" generated by the CLI program from `crowdin init` command is over 1000 characters. For this reason, could we remove this length field?